README.md

```diff
@@ -19,9 +19,10 @@ inherit_gem:
   rubocop-github:
     - config/default.yml
     - config/rails.yml
-    - config/accessibility.yml
 ```
 
+💭 Looking for `config/accessibility.yml` and the `GitHub/Accessibility` configs? [They have been moved](https://github.com/github/rubocop-rails-accessibility) to a new gem.
```

Comment:
Nit

```suggestion
💭 Looking for `config/accessibility.yml` and the `GitHub/Accessibility` configs? They have been moved to [a new gem](https://github.com/github/rubocop-rails-accessibility).
```

I'd expect the first bit to link to the PR where they were moved, or something like that, hence moving the link.